test/basic.test.js

```diff
@@ -593,19 +593,18 @@ test('children with same names render in correct order', async ({ equal }) => {
   equal(a, 3, 'last logged object takes precedence')
 })
 
-// https://github.com/pinojs/pino/pull/251 - use this.stringify
-test('use `fast-safe-stringify` to avoid circular dependencies', async ({ same }) => {
+test('use `json-stringify-safe` to avoid circular dependencies', async ({ same }) => {
```

Comment:
is the implementation detail relevant to the test? shouldn't it be about the observable behaviour?

Reply:
Unfortunately it is. See the line changed in the output.